### PR TITLE
Commerce 9047

### PR DESCRIPTION
**Pull request by MahmoudAzzam**

### CODE REVIEW

**Comment by MahmoudAzzam** [comment]
ci:test:sf

**Comment by MahmoudAzzam** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
@MahmoudAzzam looks like you need to run **baseline** too.
Also I requested @dsitu review because if I remember well in this case userid should be the first parameter in the signature of the copy method.

@alessiorendina can you have a look at this too?

Thanks all!

**Comment by MahmoudAzzam** [comment]
@andrea-ale-sbarra I though the parameters list should be in alphabetical order to conform to liferay standards. Please correct me if I am wrong

**Comment by andrea-ale-sbarra** [comment]
@MahmoudAzzam you are right but this if I remember well should go first. Let's wait for experts confirmation :)

**Comment by alessiorendina** [comment]
@MahmoudAzzam we are trying to find a pattern for this. Meanwhile, I have found examples in the code where we use groupId first only when we are reading from DB. Otherwise, we use userId first.

cc @andrea-ale-sbarra

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java [hunk 1]
```diff
@@ -515,18 +515,20 @@ public CPDefinition copyCPDefinition(long cpDefinitionId)
 			cpDefinitionId);
 
 		return cpDefinitionLocalService.copyCPDefinition(
-			cpDefinitionId, cpDefinition.getGroupId());
+			cpDefinitionId, cpDefinition.getGroupId(),
+			cpDefinition.getUserId());
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java:519]: @MahmoudAzzam @alessiorendina confirmed my doubt, so please move userid to be first param here and make baseline run

**Pull request closed without merging**